### PR TITLE
Factsheets-actors page

### PR DESCRIPTION
**Pull request by hectoruch**
Add:

image

### CODE REVIEW

**Comment by hectoruch** [comment]
Add:
- Grid lines

Delete: 
- Tick lines on chart

Update:
- Colors, font size, font family and other styles

image

File: package.json [hunk 1]
```diff
@@ -48,6 +48,17 @@
   "dependencies": {
     "body-parser": "^1.15.2",
     "d3": "^4.2.2",
+    "d3-array": "^1.0.1",
```
Comment (nerik) [inline package.json]: There is no need to manually import all d3 modules, as the entry point in the `d3` module already has references to all d3 modules.

File: scripts/components/tables/area-tables.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  .selectAll('td')
+  .data(function(d){return d;})
+  .enter().append('td')
+  .style('border', '1px black solid')
+  .style('padding', '10px')
+  // .on("mouseover", function(){d3_select(this).style("background-color", "aliceblue")})
+  // .on("mouseout", function(){d3_select(this).style("background-color", "#fff1ae")})
+  .text(function(d){return d;})
+  .style('font-size', '12px');
```
Comment (nerik) [inline scripts/components/tables/area-tables.js]: While it is technically possible to build tables in d3 I strongly advise against it. Let's just use good old html tables for this.
We have a templating engine in the project that you can use, see example here for use with a table https://github.com/Vizzuality/TRASE-frontend/pull/77/commits/be5e9cab367e6a40c94498b579e3b16e6d0b5b2d

**Review by nerik — CHANGES_REQUESTED** [review]

**Comment by hectoruch** [comment]
Add styles for table:

image

File: package.json (hunk 1 above)
Comment (agnlez) [reply package.json]: Agree. Please remove all those dependencies and use d3 modules already installed in the project. If you need any help with this, let us know.

File: html/factsheets.ejs [hunk 3]
```diff
@@ -8,7 +8,110 @@
   <div class="l-factsheets js-factsheets">
     <%= htmlWebpackPlugin.options.nav %>
```
Comment (agnlez) [inline html/factsheets.ejs]: You forgot about adding a `wrap` here

Comment (agnlez) [reply html/factsheets.ejs]: Please, add `max-width` to the layout and use the variable already declared in `_settings` file.

File: scripts/components/graphs/area-stack.component.js [hunk 4]
```diff
@@ -42,7 +42,7 @@ export default class {
 
     this.options = {
       width: this.el.clientWidth,
-      height: this.el.clientHeight
+      height: 380//this.el.clientHeight
```
Comment (agnlez) [inline scripts/components/graphs/area-stack.component.js:45]: are we sure this is not going to change?

File: scripts/components/graphs/area-stack.component.js [hunk 5]
```diff
@@ -143,6 +143,24 @@ export default class {
       g.append('g')
           .attr('class', 'axis axis--y')
           .call(d3_axis_left(y).ticks(5, 's'));
+
+      g.selectAll('.axis--y > .tick')
+        .select('line') //grab the tick line
+        .attr('x2', width)
+        .attr('stroke', '#34444c')
+        .attr('class', 'tick-line') //style with a custom class and CSS
+        .style('stroke-width', 1); //or style directly with attributes or inline styles
```
Comment (agnlez) [inline scripts/components/graphs/area-stack.component.js]: Please, move styles to its styles file.

**Comment by hectoruch** [comment]
Add icons on:
- Table 
  image
- Select area

image

File: styles/components/factsheets/area-select.scss [hunk 6]
```diff
@@ -31,10 +31,22 @@
...
       .value-information {
+        display: flex;
+        align-items: center;
         margin-top: 7px;
         font-family: $font-family-1;
         font-size: $font-size-medium;
         letter-spacing: -1px;
+
+        .star-icon {
+          width: 16px;
```
Comment (agnlez) [inline styles/components/factsheets/area-select.scss]: Icon sizes are setted in `icons` styles file. If you need to positionate them or similar do it here.

File: styles/components/factsheets/area-select.scss [hunk 7]
```diff
@@ -31,10 +31,22 @@
...
+          margin-right: 2px;
+        }
+
+        .icon-check {
+          width: 16px;
+          margin-right: 4px;
```
Comment (agnlez) [inline styles/components/factsheets/area-select.scss:46]: Same here.

File: package.json [hunk 8]
```diff
@@ -1,65 +0,0 @@
-{
```
Comment (agnlez) [inline package.json]: Restore this! :P

File: scripts/components/graphs/area-stack.component.js [hunk 9]
```diff
@@ -158,7 +158,8 @@ export default class {
       g.selectAll('.axis--x > .tick')
         .select('line') //grab the tick line
         .attr('class', 'tick-line') //style with a custom class and CSS
-        .attr('y2', 0);
+        .attr('y2', 0)
+        .style('stroke-width', 1); //or style directly with attributes or inline styles
```
Comment (agnlez) [inline scripts/components/graphs/area-stack.component.js]: Move this to its CSS file, please.

File: html/factsheets.ejs [hunk 10]
```diff
@@ -5,116 +5,118 @@
...
-          <ul class="js-dropdown-list dropdown-list">
-            <li class="js-dropdown-item dropdown-item" data-value="Soy">Soy</li>
-          </ul>
-        </div>
-        <div class="contain-information">
-          <div>
-            <span class="title-information">
-              ZERO DEFORESTATION COMMITMENT
+    <div class="l-factsheets">
+      <div class="content wrap">
```
Comment (agnlez) [inline html/factsheets.ejs]: Don't use the `wrap` with something else, if you need another class or whatever, use another different element

File: styles/layouts/l-factsheets.scss [hunk 11]
```diff
@@ -1,7 +1,9 @@
 .l-factsheets {
+  // max-width: $max-width-grid;
```
Comment (agnlez) [inline styles/layouts/l-factsheets.scss]: Get rid of commented code.

**Pull request merged**